### PR TITLE
Feature/auth

### PR DESCRIPTION
**Pull request by joejemmely**

### CODE REVIEW

**Comment by joejemmely** [comment]
@vladimirsan, we changed the secret right after it was pushed on GitHub. Don't hit me in the mouth, please 🥺

File: pages/api/login.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const microAuthGoogle = require('microauth-google');
+
+const options = {
+  clientId: process.env.CLIENT_ID,
+  clientSecret: process.env.CLIENT_SECRET,
+  callbackUrl: 'http://localhost:3000/login',
```
Comment (vladimirsan) [inline pages/api/login.js]: I'm thinking this could go into process.env too

**Comment by vladimirsan** [comment]
should this be closed? I'm a little bit confused :p

**Comment by joejemmely** [comment]
So are we 🙈it's not quite working as we'd like. We'll keep working on it tonight.

**Pull request merged**